rules-tests/Privatization/Rector/Class_/FinalizeClassesWithoutChildrenRector/Fixture/class_with_attribute.php.inc

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Rector\Tests\Php82\Rector\Class_\ReadOnlyClassRector\Fixture;
```

Comment:
oh, the namespace need to be changed based on the test namespace, I will change that :)

Reply:
ah damit, totally missed that. Thank you for taking care of that.

Reply:
https://github.com/rectorphp/rector-src/pull/4586